assets/src/hooks/usePersistedStateReducer.ts

```diff
@@ -26,9 +26,14 @@ const APP_STATE_KEY = "mbta-skate-state"
 type Key = string[]
 
 const LOCALLY_PERSISTED_KEYS: Key[] = [
+  ["notificationDrawerIsOpen"],
+  ["openView"],
+  ["pickerContainerIsVisible"],
   ["selectedShuttleRouteIds"],
   ["selectedShuttleRunIds"],
+  ["selectedVehicleOrGhost"],
```

Comment:
Just curious, how does this play along with cases where the vehicle is no longer active when the user opens the application again?